spec/controllers/users/passwords_controller_spec.rb

```diff
@@ -69,10 +69,16 @@
       end
 
       it 'creates a user Event for the password change' do
-        stub_sign_in(create(:user))
+        user = stub_sign_in
 
-        params = { password: 'salty new password' }
-        patch :update, params: { update_user_password_form: params }
+        params = {
+          password: 'salty new password',
+          password_confirmation: 'salty new password',
```

Comment:
The addition of `password_confirmation` here is due to some code drift where we added the requirement to enter the new password twice in #8220. Since we weren't verifying anything, this test never began failing when adding that requirement.